scipy/optimize/optimize.py

```diff
@@ -425,8 +424,34 @@ def _minimize_neldermead(func, x0, args=(), callback=None,
         ``initial_simplex[j,:]`` should contain the coordinates of
         the j-th vertex of the ``N+1`` vertices in the simplex, where
         ``N`` is the dimension.
-
+    xatol : float, optional
+        Absolute error in xopt between iterations that is acceptable for
+        convergence.
+    fatol : number, optional
+        Absolute error in func(xopt) between iterations that is acceptable for
+        convergence.
     """
+    if 'ftol' in unknown_options:
+        warnings.warn("ftol is deprecated for _minimize_neldermead,"
+                      " use fatol instead. If you specified both, only"
```

Comment:
`_minimize_neldermead` shouldn't be mentioned in the warning message as it's not a public function and the end user might be confused, maybe talk about the "nelder-mead" method instead.